app/views/state_file/questions/email_address/edit.html.erb

```diff
@@ -4,9 +4,11 @@
   <h1 class="h2" id="main-question"><%= title %></h1>
 
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <div class="blue-group">
```

Comment:
Formats these pages to look like figma
<img width="732" alt="Screenshot 2024-01-25 at 12 31 58 PM" src="https://github.com/codeforamerica/vita-min/assets/49880002/3659b058-bbc1-4983-8141-5472b752368b">